### PR TITLE
Virtual Keyboard

### PR DESCRIPTION
**Pull request by FrostyDog**
1. Task:
https://github.com/rolling-scopes-school/tasks/blob/master/tasks/virtual-keyboard/virtual-keyboard-en.md
2. Screenshot
image
3. Deploy
https://frostydog.github.io/virtual-keyboard/

4. Done 28.05.20 (deadline 28.05.20)
5. Score: 200 / 200
- [x] Minimal scope
    - [x] Feature 1 is implemented (20/20)
    - [x] Feature 2 is implemented (5/10)
- [x] Basic scope
    - [ ] Feature 3 is implemented (0/15)
    - [x] Feature 4 is implemented (7/15)
- [x] Extra scope
    - [x] Feature 5 is implemented (12/15)
- [x] Technical requirements
    - [x] ESLint is used (15/15)
    - [x] ES6 features are used (15/15)
    - [x] Requirements to the repository, commits and pull request are met (10/10)
- [ ] Penalties --??
    - [ ] Errors while app's working. (-10 for each, but no more than the total number of points for requirement implementation)
    - [ ] Not comply with the requirements for Pull Request, repository, commits names (-10) scores by mentor

### CODE REVIEW

**Review by kat-ya — CHANGES_REQUESTED** [review]
@FrostyDog ,
there seems to be some problem with the demo of the functionality of a keyboard on https://frostydog.github.io/virtual-keyboard/ on my computer. looks like textarea.js is missing
Screen Shot 2020-05-31 at 3 48 28 PM

can you, please, take a look at it

File: package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "url": "https://github.com/FrostyDog/virtual-keyboard/issues"
+  },
+  "homepage": "https://github.com/FrostyDog/virtual-keyboard#readme",
+  "devDependencies": {
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.2"
+  },
+  "dependencies": {
+    "lint": "^0.7.0"
```
Comment (kat-ya) [inline package.json]: are you sure `lint` should be a dependency? 
I see youhave eslint as a devDependency, so `lint` seems redundant and might increase bundle size

File: package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "virtual-keyboard",
+  "version": "1.0.0",
+  "description": "��#\u0000 \u0000v\u0000i\u0000r\u0000t\u0000u\u0000a\u0000l\u0000-\u0000k\u0000e\u0000y\u0000b\u0000o\u0000a\u0000r\u0000d\u0000\r\u0000 \u0000",
+  "main": "index.js",
+  "scripts": {
+    "test": "npm run lint",
+    "lint": "node node_modules/eslint/bin/eslint.js ./modules"
```
Comment (kat-ya) [inline package.json]: usually once you install a package like
`npm i --save-dev eslint`
you can use it in `scripts` like
`eslint src`

**Review by kat-ya — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (FrostyDog) [reply package.json]: Perfectly true, fixed. Thanks!

File: package.json (hunk 2 above)
Comment (FrostyDog) [reply package.json]: Understood, did not had a `eslint-cli installed` , so `eslint` coomand was not recognizable.
Fixed!3

**Comment by FrostyDog** [comment]
> @FrostyDog ,
> there seems to be some problem with the demo of the functionality of a keyboard on https://frostydog.github.io/virtual-keyboard/ on my computer. looks like textarea.js is missing
> Screen Shot 2020-05-31 at 3 48 28 PM
> 
> can you, please, take a look at it

Took a look and fixed. Paths were wrong, I suppose.
Updated gh-pages and dev.
Thank you for checking ! :)

**Review by kat-ya — APPROVED** [review]

**Pull request merged**